src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/ComTypes/IEnumFormatETC.cs

```diff
@@ -24,7 +24,7 @@ public interface IEnumFORMATETC
         ///     in NULL for that parameter).
         /// </summary>
         [PreserveSig]
-        int Next(int celt, [Out, MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 0)] FORMATETC[] rgelt, [Out, MarshalAs(UnmanagedType.LPArray)] int[] pceltFetched);
+        int Next(int celt, [Out, MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 0)] FORMATETC[] rgelt, [Out, MarshalAs(UnmanagedType.LPArray)] int[]? pceltFetched);
```

Comment:
[ref](https://github.com/dotnet/runtime/blob/021ea49994f7f472f8f8057ecef5faed28ea9a16/src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs#L1736) also needs an update?

Reply:
Yes. I'd also like to know if we plan on updating more COM interfaces. The issue here is although it is likely okay, it is also an annoyance that may negatively impact scenarios for users that aren't very interested in anything more than moving up to .NET latest. These sorts of changes could force people to go into their "legacy" COM code and be forced to update. We've received occasional push back on changing much in this area.

I think I'd like to see us go through all the COM interfaces and update as appropriate or just leave them as-is. Fixing them in a piece meal fashion is far more annoying than helpful in my opinion.

Reply:
I can left this PR hanging until I do more WinForms work. I honestly work in peice meal fashion, and cannot catch this one problem by manually inspecting code. After I'm done with WinForms I can get list of interfaces which affects me and then you can decide.

Reply:
Thanks @kant2002. I'm going to switch this to draft in the meantime.